### DISCUSSION TITLE
store data in a ulg file

### DISCUSSION THREAD

**Discussion by jsandlerus** [npm/feedback#406] [General]
Hi, I'm trying to store JavaScript data into a ulg file extension. I've been reviewing the documentation but I haven't found if this feature is actually available or not. I don't know if it has something to do with Debug?

I need to create a ulg file and then store some JavaScript data in it, I have to convert this data in ulg also.

**Comment by MylesBorins** [comment]
Hey @jsandleraol . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
